src/intervals/interval_operations/constants.jl

```diff
@@ -41,7 +38,9 @@ of `Interval` for more information about the default interval falvor.
 
 Implement the `entire` function of the IEEE Std 1788-2015 (section 10.5.2).
 """
-entireinterval(::Type{F}) where {T, F<:Interval{T}} = F(-Inf, Inf)
-entireinterval(::F) where {F<:Interval} = entireinterval(F)
-entireinterval(::Type{T}) where T = Interval{T}(-Inf, Inf)
+entireinterval(::Type{F}) where {T<:NumTypes,F<:Interval{T}} = F(typemin(T), typemax(T))
+entireinterval(::Type{T}) where {T<:NumTypes} = entireinterval(Interval{T})
```

Comment:
i think these methods need to be more constrained, separating rational intervals from those with floats.

Reply:
How come? Is this related to https://github.com/JuliaIntervals/IntervalArithmetic.jl/pull/559#discussion_r1168082867?
On my machine, I have
```julia
julia> entireinterval(Interval{Rational{Int}})
[-1//0, 1//0]
```